statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -109,9 +109,15 @@ func (psh *PresenterStatusHandler) CalculateRewardsPerHour() string {
 	rewardsPerHourCoefficient := chanceToBeInConsensus * roundsPerHourAccordingToHitRate
 	totalRewardsPerHourFloat := big.NewFloat(rewardsPerHourCoefficient)
 	totalRewardsPerHourFloat.Mul(totalRewardsPerHourFloat, rewardsInErd)
+	return totalRewardsPerHourFloat.Text('f', precisionRewards)
+}
 
-	totalRewardsPerHour := new(big.Int)
-	totalRewardsPerHourFloat.Int(totalRewardsPerHour)
+// GetZeros will return a string with a specific number of zeros
+func (psh *PresenterStatusHandler) GetZeros() string {
+	retValue := "." + strings.Repeat("0", precisionRewards)
```

Comment:
Why not first test if precisionRewards is 0 and then early exit with `return ""` ?

Reply:
I get a warning because precisionRewards will be never zero.

Reply:
ok